packages/admin/README.md

```diff
@@ -63,10 +63,18 @@ The SDK will also provide TS typings out of the box. Please, refer to the implem
 
 ## Developer commands
 
+### `derive-airnode-xpub`
+
+Derives the Airnode extended public key (xpub). Many of the CLI commands require the xpub as a parameter.
```

Comment:
I'm not sure if "many" is right here since only a single command actually requires it (`derive-sponsor-wallet-address`)